cmake/modules/AddSwift.cmake

```diff
@@ -744,7 +745,7 @@ function(add_swift_host_library name)
   _set_target_prefix_and_suffix(${name} "${libkind}" "${SWIFT_HOST_VARIANT_SDK}")
 
   if (ASHL_SHARED AND ASHL_HAS_SWIFT_MODULES)
-      _add_swift_runtime_link_flags(${name} "." "")
+      _add_swift_runtime_link_flags(${name} "." "" FALSE)
```

Comment:
I don't think any shared (`ASHL_SHARED`) libraries are required for building stdlib.

Reply:
Ah this is only shared, true. Having said that, if we did move to shared libraries more, it would be the case that they are "required for minimal" but this would still be FALSE because they wouldn't have any Swift. And that was more my point.